src/main/java/by/exadel/internship/service/impl/FormServiceImpl.java

```diff
@@ -62,7 +68,14 @@ public FormFullDTO process(FormRegisterDTO form, MultipartFile file) {
 
     private FormFullDTO saveForm(FormRegisterDTO formRegisterDTO) {
 
+        Country country = countryRepository.findByName(formRegisterDTO.getCountry().getName());
```

Comment:
If you try to find by name it should cover exception cases (like Not found).
Question: Why can't UI send country/city id?

Reply:
Done,thanks
Question: Why can't UI send country/city id? -> they can, and the will. I talked with them.